marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubSecurityAdminTest.java

```diff
@@ -271,4 +272,28 @@ void task5CreateRoleWithCustomPrivilegeInDisallowedNamespace() {
             new Role(adminUserApi, roleName).delete();
         }
     }
+
+    @Test
+    void CreateCustomRoleWithXdmpShutdown() {
+        MarkLogicVersion version = new MarkLogicVersion(getHubClient().getManageClient());
+        Assumptions.assumeTrue(version.getMajor() >= 10 &&
+            (version.isNightly() || version.getMinor() >= 601), "Bug<55765> is fixed in 10.0-6.1");
+
+        final String roleName = "custom-role-with-xdmp-shutdown-privilege";
+        Role customRole = new Role(userWithRoleBeingTestedApi, roleName);
+        customRole.addPrivilege(new RolePrivilege("xdmp:shutdown", "http://marklogic.com/xdmp/privileges/xdmp-shutdown", "execute"));
+
+        try {
+            customRole.save();
+            fail("The role creation should have failed because a DHSA user shouldn’t be able to create a custom role with xdmp:shutdown privilege");
```

Comment:
This is good - assertion messages are critical because they explain why something is expected (an assertion message that just restates what the assertion itself is stating doesn't add any value).